### PR TITLE
Integrate Blockscout Blockchain Explorer

### PR DESCRIPTION
**Pull request by akarabashov**
Tasks:

- [x] Integrate Blockscout Blockchain Explorer
- [x] Customize the UI and remove unnecessary fields
~~Fix smart contract transactions decoding~~

### CODE REVIEW

File: indy-besu/network/infra/blockscout/envs/common-blockscout.env [hunk 1]
```diff
@@ -0,0 +1,278 @@
+# DOCKER_TAG=
+ETHEREUM_JSONRPC_VARIANT=geth
+ETHEREUM_JSONRPC_HTTP_URL=http://host.docker.internal:8545/
```
Comment (Artemkaaas) [inline indy-besu/network/infra/blockscout/envs/common-blockscout.env:2]: Please clean this file out of unused variables

File: indy-besu/network/infra/blockscout/envs/common-frontend.env [hunk 2]
```diff
@@ -0,0 +1,17 @@
+NEXT_PUBLIC_API_HOST=localhost:26000
+NEXT_PUBLIC_API_PROTOCOL=http
+NEXT_PUBLIC_STATS_API_HOST=http://localhost:26001
+NEXT_PUBLIC_NETWORK_NAME=Awesome chain
```
Comment (Artemkaaas) [inline indy-besu/network/infra/blockscout/envs/common-frontend.env]: Should it be Indy-Besu?

File: indy-besu/network/infra/blockscout/envs/common-blockscout.env (hunk 1 above)
Comment (akarabashov) [reply indy-besu/network/infra/blockscout/envs/common-blockscout.env:2]: Done

File: indy-besu/network/infra/blockscout/envs/common-frontend.env (hunk 2 above)
Comment (akarabashov) [reply indy-besu/network/infra/blockscout/envs/common-frontend.env]: Done

**Pull request merged**